limesurvey/limesurvey.py

```diff
@@ -148,11 +149,37 @@ def get_student_access_code(self, anonymous_user_id):
 
         return response.get("result").get("token")
 
-    def add_participant_to_survey(self, user, anonymous_user_id):
+    @staticmethod
+    def get_fullname(user) -> Tuple[str, str]:
+        """
+        Return the full name of the user.
+
+        args:
+            user: The user to get the fullname
+
+        returns:
+            A tuple containing the first name and last name of the user
+        """
+        first_name, last_name = "", ""
+
+        if user.profile.name:
+            fullname = user.profile.name.split(" ", 1)
+            first_name = fullname[0]
+
+            if fullname[1:]:
+                last_name = fullname[1]
```

Comment:
```suggestion
       if fullname[1:]:
            last_name = fullname[1]
```
It reads better when we have 1 level of nested instead of 2

Reply:
The problem is that if the `user.profile.name` variable is `None`, the code fails.

```python
if fullname[1:]:
NameError: name 'fullname' is not defined
```